src/figanos/matplotlib/plot.py

```diff
@@ -2023,15 +2024,16 @@ def taylordiagram(
 
         ax.clabel(ct, ct.levels, fontsize=8)
 
-        ct_line = Line2D(
+        # points.append(ct_line)
+        ct_line = ax.plot(
             [0],
             [0],
             ls=contours_kw["linestyles"],
             lw=1,
             c="k" if "colors" not in contours_kw else contours_kw["colors"],
             label="rmse",
         )
-        points.append(ct_line)
+        points.append(ct_line[0])
```

Comment:
without this change, the text appears on the legend, but weirdly, it can't be accessed with methods like `.get_legend_handles_labels`